SampleSPM/SampleSPM.xcodeproj/project.pbxproj

```diff
@@ -1034,6 +1070,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = 575EYS6ZJA;
 				INFOPLIST_FILE = "$(SRCROOT)/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
So you bumped target version of Sample app? (and you still use @available in the code?)

Reply:
This is on `SampleSPM.xcodeproj`, there's no issue when we open the project based on this file.

When we open the file from `RInAppMessaging.xcodeproj`, we need to add `@available` to check the os version.